File: MYCM/backend/utils/SimpleJSONMessageService.cs
namespace backend.utils{
    /// <summary>
    /// Simples service for representing messages in JSON
    /// </summary>
    public sealed class SimpleJSONMessageService{
        /// <summary>
        /// String with the message
        /// </summary>
        public string message{get;set;}
        /// <summary>
        /// Builds a new JSONMessageService with the message being represented in JSON
        /// </summary>
        /// <param name="message">String with the message</param>
        public SimpleJSONMessageService(string message){this.message=message;}
    }
}

File: MYCA/myca/webservices/src/main/java/cdiomyc/webservices/dataservices/json/SimpleJSONMessageService.java
package cdiomyc.webservices.dataservices.json;

/**
 * Service for representing simple JSON messages
 * @author <a href="https://github.com/freitzzz">freitzzz</a>
 */
public final class SimpleJSONMessageService {
    
    /**
     * String with the message being represented
     */
    public String message;
    
    /**
     * Builds a new SimpleJSONMessageService
     * @param message String with the message being represented
     */
    public SimpleJSONMessageService(String message){this.message=message;}
}
